src/main/java/com/carhouse/provider/impl/CarMakeProviderImpl.java

```diff
@@ -34,8 +34,12 @@
      * @return the car make
      */
     public CarMake getCarMake(final String carMakeId) {
-        if (carMakeId != null && Integer.parseInt(carMakeId) >= 0) {
+        if (carMakeId != null && !carMakeId.isEmpty()) {
+            try {
             return new CarMake(Integer.parseInt(carMakeId), "BMW");
+            } catch (NumberFormatException ex) {
```

Comment:
looks strange.
Are you pretty confident that it is the best way how to validate `carMakeId`?